src/test/ui/consts/const-eval/ub-incorrect-vtable.64bit.stderr

```diff
@@ -0,0 +1,52 @@
+error: any use of this value will cause an error
+  --> $DIR/ub-incorrect-vtable.rs:19:14
+   |
+LL | / const INVALID_VTABLE_ALIGNMENT: &dyn Trait =
+LL | |     unsafe { std::mem::transmute((&92u8, &[0usize, 1usize, 1000usize])) };
+   | |______________^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^__-
+   |                |
+   |                invalid vtable: alignment `1000` is not a power of 2
+   |
+   = note: `#[deny(const_err)]` on by default
+   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
+   = note: for more information, see issue #71800 <https://github.com/rust-lang/rust/issues/71800>
+
+error: any use of this value will cause an error
+  --> $DIR/ub-incorrect-vtable.rs:25:14
+   |
+LL | / const INVALID_VTABLE_SIZE: &dyn Trait =
+LL | |     unsafe { std::mem::transmute((&92u8, &[1usize, usize::MAX, 1usize])) };
+   | |______________^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^__-
+   |                |
+   |                invalid vtable: size is bigger than largest supported object
+   |
+   = warning: this was previously accepted by the compiler but is being phased out; it will become a hard error in a future release!
+   = note: for more information, see issue #71800 <https://github.com/rust-lang/rust/issues/71800>
+
+error[E0080]: it is undefined behavior to use this value
+  --> $DIR/ub-incorrect-vtable.rs:36:1
+   |
+LL | / const INVALID_VTABLE_ALIGNMENT_UB: W<&dyn Trait> =
+LL | |     unsafe { std::mem::transmute((&92u8, &(drop_me as fn(*mut usize), 1usize, 1000usize))) };
+   | |_____________________________________________________________________________________________^ type validation failed: encountered invalid vtable: alignment `1000` is not a power of 2 at .0
+   |
+   = note: The rules on what exactly is undefined behavior aren't clear, so this check might be overzealous. Please open an issue on the rustc repository if you believe it should not be considered undefined behavior.
+   = note: the raw bytes of the constant (size: 16, align: 8) {
+               ╾───────allocN───────╼ ╾───────allocN───────╼ │ ╾──────╼╾──────╼
+           }
+
+error[E0080]: it is undefined behavior to use this value
+  --> $DIR/ub-incorrect-vtable.rs:41:1
+   |
+LL | / const INVALID_VTABLE_SIZE_UB: W<&dyn Trait> =
+LL | |     unsafe { std::mem::transmute((&92u8, &(drop_me as fn(*mut usize), usize::MAX, 1usize))) };
+   | |______________________________________________________________________________________________^ type validation failed: encountered invalid vtable: size is bigger than largest supported object at .0
```

Comment:
@oli-obk reading this, I feel like we should reorder these error messages a bit, and make them more like
> type validation failed at .0: encountered invalid vtable: size is bigger than largest supported object

What do you think? (@lqd certainly not for this PR though)

Reply:
if you and oli want this, I can certainly do that in this PR

Reply:
This will affect many tests, so it should be a separate PR. But sure, if you want to help wit this that'd be great. :)

Reply:
Agreed, I'll do that in another PR.

Would something like this work for you ? Changing `ValidationFailure`'s content from a String message to a `path: Option<String>, mesg: String` so that https://github.com/rust-lang/rust/blob/3e827cc21e0734edd26170e8d1481f0d66a1426b/compiler/rustc_mir/src/interpret/validity.rs#L27-L47 can handle the empty "where", and the formatting you want can be implemented at https://github.com/rust-lang/rust/blob/2779fc1c4768567b74b757a8083db3c39765f071/compiler/rustc_middle/src/mir/interpret/error.rs#L326

Reply:
Let's discuss the implementation details elsewhere (possibly on Zulip), if/when @oli-obk agrees that we even want to do this. :)

Reply:
I agree that the proposed ordering is better than the current one